example/30-infrastructure.yaml

```diff
@@ -59,6 +59,9 @@ spec:
     # internal: 10.243.0.0/19
     # cloudNAT:
     #   minPortsPerVM: 2048
+    #   natIPNames:
```

Comment:
This lacks the example for IPs instead of names.

Reply:
We only allow names here, because we require the unique name for the data source.